multi-node/vagrant/turbo-server-config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "turbo-server-address": "1.2.3.4",
```

Comment:
make sense. To make it consistent do we want user to specify port number for turbo server?